lightdash.yml

```diff
@@ -1,7 +1,3 @@
 version: '1.0'
 
-projects:
-  - name: default
-    type: 'dbt'
-    profiles_dir: /usr/app/profiles
-    project_dir: /usr/app/dbt
```

Comment:
We need to keep this since projects is required. There is a ticket to fix it: https://github.com/lightdash/lightdash/issues/520

Reply:
Whoops! Didn't mean to include that, thanks!